docs/cluster-configuration.md

```diff
@@ -897,6 +897,21 @@ spec:
 All the values defined [here](https://kubevirt.io/api-reference/master/definitions.html#_v1_logverbosity)
 can be applied.
 
+### Containerized data importer (CDI)
+Different levels of verbosity are used in CDI to control the amount of information that is logged. The verbosity level of logs in CDI can be adjusted using a dedicated field that affect all of its components. 
+This feature enables users to control the amount of detail displayed in logs, ranging from minimal to detailed debugging information.
+
+For example:
+```yaml
+kind: HyperConverged
+metadata:
+  name: kubevirt-hyperconverged
+spec:
+  logVerbosityConfig:
+    cdi:
+      logVerbosity: 4
+```
```

Comment:
According to the new API that defined above, there is no nested field `logVerbosity` within the `cdi` field.
I think it should be
```suggestion
```yaml
kind: HyperConverged
metadata:
  name: kubevirt-hyperconverged
spec:
  logVerbosityConfig:
    cdi: 4
```
```

Reply:
Right, thanks for the clarification!

Reply:
Thanks for fixing. But if the values are 1-3 - please don't use 4 (-:

Reply:
I changed it to 3, the 4 is from the old comment.